### PR TITLE
Bug 882140 - [calendar] progress spinner: Text is too long

### PR DESCRIPTION
**Pull request by orvi2014**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: apps/calendar/style/modify_account_view.css [hunk 1]
```diff
@@ -24,7 +24,7 @@
 }
 
 #modify-account-view.in-progress .during-progress > div {
-  display: inline-block;
+
```
Comment (KevinGrandon) [inline apps/calendar/style/modify_account_view.css]: nit: remove this empty line?

**Pull request closed without merging**